src/lib/change-email-helpers.js

```diff
@@ -0,0 +1,118 @@
+const helpers = require('../lib/helpers');
+const uuid = require('uuid/v4');
+const moment = require('moment');
+const { pool } = require('../lib/connectors/db');
+const DB = require('./connectors/db');
+const { EmailChangeError } = require('../controllers/change-email');
+const Repository = require('@envage/hapi-pg-rest-api/src/repository');
+
+class UsersRespository extends Repository {
```

Comment:
I think we should have each repository class defined in their own file, and most of the functions below could be methods on those classes where they relate mostly to the data access layer, like you've done with `findById` and `checkEmailAddress`